src/xcode/ENA/ENA/Resources/Localization/ar.lproj/Localizable.strings

```diff
@@ -461,6 +461,10 @@
 
 "ExposureSubmissionResult_testInvalidDesc" = "هناك مشكلة في تقييم اختبارك. الرجاء الاتصال بدائرة الصحة العامة لمعرفة كيفية المتابعة.";
 
+"ExposureSubmissionResult_testRedeemed" = "Test has errors";
+
+"ExposureSubmissionResult_testRedeemedDesc" = "There was a problem evaluating your test. Your QR Code has already expired.";
```

Comment:
Hi @steins-code 
Is it against the translation process?

Reply:
What is the exact issue because we have to create placeholder in all language files?

Reply:
We have some internal process/agreement with translation team. @steins-code will inform them and I'll inform QA lead to avoid some unnecessary bug tickets.

Reply:
Talked to translation colleagues: Sabine Loss and Gudrun Wolber.
Outcome was that we only need the new text in german.